src/python_testing/TC_CC_10_1.py

```diff
@@ -0,0 +1,484 @@
+        asserts.assert_equal(result.status, Status.Success, "Store Scene failed on status")
+        asserts.assert_equal(result.groupID, self.kGroup1, "Store Scene failed on groupID")
+        asserts.assert_equal(result.sceneID, 0x01, "Store Scene failed on sceneID")
+
+        self.step("4")
+        result = await self.TH1.SendCommand(self.dut_node_id, 1, Clusters.ScenesManagement.Commands.ViewScene(self.kGroup1, 0x01))
+        asserts.assert_equal(result.status, Status.Success, "View Scene failed on status")
+        asserts.assert_equal(result.groupID, self.kGroup1, "View Scene failed on groupID")
+        asserts.assert_equal(result.sceneID, 0x01, "View Scene failed on sceneID")
+        asserts.assert_equal(result.transitionTime, 0, "View Scene failed on transitionTime")
+
+        for EFS in result.extensionFieldSets:
+            if EFS.clusterID == 0x0300:
+                for AV in EFS.attributeValueList:
+                    if AV.attributeID == 0x0001 and self.pics_guard(self.check_pics("CC.S.C06.Rsp")):
+                        asserts.assert_less_equal(AV.valueUnsigned8, 58, "View Scene failed on Satuation above limit")
+                        asserts.assert_greater_equal(AV.valueUnsigned8, 42, "View Scene failed on Satuation below limit")
+
+                    if AV.attributeID == 0x0003 and self.pics_guard(self.check_pics("CC.S.C07.Rsp")):
+                        asserts.assert_less_equal(AV.valueUnsigned16, 35000, "View Scene failed on CurrentX above limit")
+                        asserts.assert_greater_equal(AV.valueUnsigned16, 31000, "View Scene failed on CurrentX below limit")
+
+                    if AV.attributeID == 0x0004 and self.pics_guard(self.check_pics("CC.S.C07.Rsp")):
+                        asserts.assert_less_equal(AV.valueUnsigned16, 21000, "View Scene failed on CurrentY above limit")
+                        asserts.assert_greater_equal(AV.valueUnsigned16, 17000, "View Scene failed on CurrentY below limit")
+
+                    if AV.attributeID == 0x0007 and self.pics_guard(self.check_pics("CC.S.C0a.Rsp")):
+                        asserts.assert_less_equal(AV.valueUnsigned16, ColorTempPhysicalMaxMiredsValue,
+                                                  "View Scene failed on ColorTemperatureMireds above limit")
+                        asserts.assert_greater_equal(AV.valueUnsigned16, ColorTempPhysicalMinMiredsValue,
+                                                     "View Scene failed on ColorTemperatureMireds below limit")
+
+                    if AV.attributeID == 0x4000 and self.pics_guard(self.check_pics("CC.S.C43.Rsp")):
+                        asserts.assert_less_equal(AV.valueUnsigned16, 21800, "View Scene failed on EnhancedHue above limit")
+                        asserts.assert_greater_equal(AV.valueUnsigned16, 18200, "View Scene failed on EnhancedHue below limit")
```

Comment:
if you get back an attributeID that wasn't checked by any of the above conditions, is that also a failure?

Reply:
No it isn't. The color loop attributes are expected to be in there. I just didn't add a colorloop command since an active color loop would modify the enhanced hue so it was more convenient to have color loops commands at the end of the test.